File: codeforces/14.soldier_and_bananas_[546A]/sab.py
# Author: José Rodolfo (jric2002)
input_data = input().split(" ")
k = int(input_data[0])
n = int(input_data[1])
w = int(input_data[2])
borrowed_money = int((k * ((w * (w + 1)) / 2)) - n)
borrowed_money = borrowed_money if (borrowed_money > 0) else 0
print(borrowed_money)

File: codeforces/low_difficulty/difficulty_800/14.soldier_and_bananas_[546A]/sab.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned int k, n, w;
  signed int borrowed_money = 0;
  cin >> k >> n >> w;
  borrowed_money = (k * ((w * (w + 1)) / 2)) - n;
  borrowed_money = (borrowed_money > 0) ? borrowed_money : 0;
  cout << borrowed_money << endl;
  return 0;
}